ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala

```diff
@@ -96,14 +96,14 @@ package object v1 {
     *
     * See the Contract Id specification for more detail daml-lf/spec/contract-id.rst
     */
-  type SubmittedTransaction = Tx.SubmittedTransaction
+  type SubmittedTransaction = transaction.Transaction.SubmittedTransaction
```

Comment:
If you imported `com.daml.lf.transaction.Transaction` you could remove the prefix here.